code/src/WijmoProvider/Features/CellStyle.ts

```diff
@@ -76,17 +82,32 @@ namespace WijmoProvider.Feature {
             );
         }
 
-        public removeAllClasses(rowNumber: number, binding: string): void {
+        public removeAllClasses(
+            rowNumber: number,
+            binding: string,
+            refresh = false
+        ): void {
             this.getMetadata(rowNumber).removeAllClasses(binding);
+
+            // we only want to force refresh on specific cases, as this could cause infinite renders on grid
```

Comment:
How did we never caught that before?

Reply:
We had a forum post opened in September, but we never fixed it.

Reply:
Next time, separate PR :)